src/provider/neoRecommendTransform.js

```diff
@@ -0,0 +1,50 @@
+
+'use strict';
+import genericTransformer from '../genericTransformer.js';
+import recommendClient from '../services/Recommendations/Recommendation.client.js';
+import lodash from 'lodash';
+
+function requestTransform(request, context) {
+  const filter = context.libdata.config.provider.services.recommend.filters || ['rec.collectionIdentifier:' + (context.libdata.libraryId || '716500') + '-katalog'];
```

Comment:
This will fail if any of context/.libdata/.config/.provider/.services/.recommend is undefined